### DISCUSSION TITLE
resolution/4k.jpg font and background asset?

### DISCUSSION THREAD

**Discussion by miaLDN** [meisnate12/Plex-Meta-Manager-Configs#667] [General]
Kometa repository `Default-Images` has collection posters for resolution.  There also exist a Photoshop template for generating posters.

What I'm looking for, are (which are not in the photoshop template):

1. the purple background colors and the gradient used
2. the font used

`resolution/4k.jpg`

I found a purple "4K HDR" poster, perhaps in bullmoose's files, but I can't find it again.

I'm looking to make for example "4K SDR" etc.

**Comment by YozoraXCII** [comment]
Hi @planetrocky 

The font used is Comfortaa Medium,

The colour I believe is #8A46CF

The defaults are now all automatically created by a powershell script, which can be found here: https://github.com/bullmoose20/Plex-Stuff/tree/main/create_defaults

**Reply by miaLDN** [reply]
Hi,

Thanks, I will check the script :)

I’m not sure that it’s Comfortaa Medium, as the “4” is a different form, and the font is squarer, Comfortaa is more rounded.

The “4” is more like a DIN style font.

**Reply by YozoraXCII** [reply]
> Hi,
> 
> Thanks, I will check the script :)
> 
> I’m not sure that it’s Comfortaa Medium, as the “4” is a different form, and the font is squarer, Comfortaa is more rounded.
> 
> The “4” is more like a DIN style font.

Are you looking for the text for the collection or the overlay?

Try Bebas Neue Regular or Inter Medium

**Reply by miaLDN** [reply]
Thanks, I will check those. I haven’t had time to check the PowerShell script yet - will do after work :)

The collection text for the resolution series, like “4K”, “1080p”, etc.

I want to make 4K HDR, 4K SDR, etc collections.  :)

I just like all the fonts to match, OCD ;)

**Comment by miaLDN** [comment]
Thanks.  I've been looking through bullmoose's repo, and it's very comprehensive.

The PoweShell script function `CreateResolution` (snippet below), seems to indicate `ComfortAa Medium` font is used.  However that doesn't match the images in the repo for the resolution posters.  Look at the style of the "4" for quick reference.


`create_defaults/logos_resolution/4K.png`  [4K.png](https://raw.githubusercontent.com/bullmoose20/Plex-Stuff/main/create_defaults/logos_resolution/4K.png)
![4K-1](https://github.com/Kometa-Team/Community-Configs/assets/33760644/68ddaeb0-c7d6-4d80-a1a1-2dc382ec6f10)


`Comfortaa Medium.ttf` from the repo:
![Screenshot 2024-07-03 175004](https://github.com/Kometa-Team/Community-Configs/assets/33760644/bf956ff5-91f9-420f-ac2f-8555966157d0)


The closest font I can find is `DIN 2014 Demi` from Adobe font collection:
![Screenshot 2024-07-03 180126](https://github.com/Kometa-Team/Community-Configs/assets/33760644/c5ed593a-224f-4d34-8ca8-ba5b6294712e)


``` ps1
################################################################################
# Function: CreateResolution
# Description:  Creates Resolution
################################################################################
Function CreateResolution {
    Write-Host "Creating Resolution"
    Set-Location $script_path
    # Find-Path "$script_path\resolution"

#[SNIP]

    $myArray = @(
        'key_name| logo| logo_offset| logo_resize| text_offset| font| font_size| font_color| border| border_width| border_color| avg_color_image| out_name| base_color| gradient| clean| avg_color| white_wash',
        'resolutions_other| transparent.png| +0| 1600| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | other| #FF2000| 1| 1| 0| 1'
    ) | ConvertFrom-Csv -Delimiter '|'

#[SNIP]

    $myArray = @(
        'key_name| logo| logo_offset| logo_resize| text_offset| font| font_size| font_color| border| border_width| border_color| avg_color_image| out_name| base_color| gradient| clean| avg_color| white_wash',
        '| 4K.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 4k| #8A46CF| 1| 1| 0| 1',
        '| 8K.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 8k| #95BCDC| 1| 1| 0| 1',
        '| 144p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 144| #F0C5E5| 1| 1| 0| 1',
        '| 240p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 240| #DFA172| 1| 1| 0| 1',
        '| 360p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 360| #6D3FDC| 1| 1| 0| 1',
        '| 480p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 480| #3996D3| 1| 1| 0| 1',
        '| 576p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 576| #DED1B2| 1| 1| 0| 1',
        '| 720p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 720| #30DC76| 1| 1| 0| 1',
        '| 1080p.png| +0| 1800| +0| ComfortAa-Medium| | #FFFFFF| 0| 15| #FFFFFF| | 1080| #D60C0C| 1| 1| 0| 1'
    ) | ConvertFrom-Csv -Delimiter '|'
```
